ci/slt/slt.sh

```diff
@@ -34,3 +34,8 @@ sqllogictest --auto-index-tables \
     -v "$@" \
     test/sqllogictest/sqlite/test \
     | tee -a target/slt.log
+
+sqllogictest --persisted-introspection \
+    -v "$@" \
+    test/sqllogictest/introspection/*.slt \
+    | tee -a target/slt.log
```

Comment:
The SLT CI job times out (as it does on main) and doesn't get to run this part (https://buildkite.com/materialize/sql-logic-tests/builds/5443). I tested it successfully locally though.